### PR TITLE
Qmk and via Swiss support

### PR DESCRIPTION
**Pull request by Bregoliii**
## Added QMK and Via Support

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ x] Keyboard (addition or update)
- [x ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/swiss/config.h
Comment (waffle87) [inline keyboards/swiss/config.h:1]: This file should be removed.

File: keyboards/swiss/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,22 @@
+//ooga booga
```
Comment (waffle87) [inline keyboards/swiss/keymaps/default/keymap.c:1]:
```suggestion
// Copyright 2024 QMK
// SPDX-License-Identifier: GPL-2.0-or-later
```

File: keyboards/swiss/keymaps/default/readme.md
Comment (waffle87) [inline keyboards/swiss/keymaps/default/readme.md:1]: This file should be removed.

File: keyboards/swiss/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,22 @@
+//ooga booga
```
Comment (waffle87) [inline keyboards/swiss/keymaps/via/keymap.c:1]:
```suggestion
// Copyright 2024 QMK
// SPDX-License-Identifier: GPL-2.0-or-later
```

File: keyboards/swiss/keymaps/via/readme.md
Comment (waffle87) [inline keyboards/swiss/keymaps/via/readme.md:1]: This file should be removed.

File: keyboards/swiss/readme.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Swiss
+
+![Swiss](https://imgur.com/8G6MB5D)
```
Comment (waffle87) [inline keyboards/swiss/readme.md:3]:
```suggestion
![Swiss](https://i.imgur.com/8G6MB5Dh.png)
```

File: keyboards/swiss/readme.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+* Keyboard Maintainer: [Bregoli](https://geekhack.org/index.php?topic=119509.0)
+* Hardware Supported: Custom HHKB PCB and C3 Unified Daughterboard
+
+Make example for this keyboard (after setting up your build environment):
+
+    make Swiss:default
+
+Flashing example for this keyboard:
+
+    make Swiss:default:flash
```
Comment (waffle87) [inline keyboards/swiss/readme.md:17]:
```suggestion
    make swiss:default

Flashing example for this keyboard:

    make swiss:default:flash
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

**Comment by Bregoliii** [comment]
uuh think i did it all correctly

**Review by dunk2k — CHANGES_REQUESTED** [review]
- add Community Layout support
  - adjust keymap layout macro name to match
- add End of File where missing

**Review by waffle87 — APPROVED** [review]

**Review by dunk2k — APPROVED** [review]

File: keyboards/swiss/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+	KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, MO(1),
+	KC_LGUI, KC_LALT,                            KC_SPC,                                      KC_RALT,          KC_RCTL
+    ),
+    /* FN */
+    [1] = LAYOUT_60_hhkb(
+        KC_GRV,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_INS,  KC_DEL,
+	KC_CAPS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_PSCR, KC_SLCT, KC_PAUS, KC_UP,   KC_TRNS, KC_TRNS,
+	KC_TRNS, KC_VOLD, KC_VOLU, KC_MUTE, KC_EJCT, KC_TRNS, KC_PAST, KC_PSLS, KC_HOME, KC_PGUP, KC_LEFT, KC_RGHT, KC_TRNS,
+	KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_PPLS, KC_PMNS, KC_END,  KC_PGDN, KC_DOWN, KC_TRNS, KC_TRNS,
+	KC_TRNS, KC_TRNS,                            KC_TRNS,                                     KC_TRNS,          KC_TRNS
```
Comment (drashna) [inline keyboards/swiss/keymaps/default/keymap.c:20]:
```suggestion
    KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSPC,
    KC_LCTL, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,
    KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, MO(1),
    KC_LGUI, KC_LALT,                            KC_SPC,                                      KC_RALT,          KC_RCTL
    ),
    /* FN */
    [1] = LAYOUT_60_hhkb(
        KC_GRV,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_INS,  KC_DEL,
    KC_CAPS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_PSCR, KC_SLCT, KC_PAUS, KC_UP,   KC_TRNS, KC_TRNS,
    KC_TRNS, KC_VOLD, KC_VOLU, KC_MUTE, KC_EJCT, KC_TRNS, KC_PAST, KC_PSLS, KC_HOME, KC_PGUP, KC_LEFT, KC_RGHT, KC_TRNS,
    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_PPLS, KC_PMNS, KC_END,  KC_PGDN, KC_DOWN, KC_TRNS, KC_TRNS,
    KC_TRNS, KC_TRNS,                            KC_TRNS,                                     KC_TRNS,          KC_TRNS
```

File: keyboards/swiss/keymaps/via/keymap.c [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+	KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, MO(1),
+	KC_LGUI, KC_LALT,                            KC_SPC,                                      KC_RALT,          KC_RCTL
+    ),
+    /* FN */
+    [1] = LAYOUT_60_hhkb(
+        KC_GRV,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_INS,  KC_DEL,
+	KC_CAPS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_PSCR, KC_SLCT, KC_PAUS, KC_UP,   KC_TRNS, KC_TRNS,
+	KC_TRNS, KC_VOLD, KC_VOLU, KC_MUTE, KC_EJCT, KC_TRNS, KC_PAST, KC_PSLS, KC_HOME, KC_PGUP, KC_LEFT, KC_RGHT, KC_TRNS,
+	KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_PPLS, KC_PMNS, KC_END,  KC_PGDN, KC_DOWN, KC_TRNS, KC_TRNS,
+	KC_TRNS, KC_TRNS,                            KC_TRNS,                                     KC_TRNS,          KC_TRNS
```
Comment (drashna) [inline keyboards/swiss/keymaps/via/keymap.c:20]:
```suggestion
    KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSPC,
    KC_LCTL, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,
    KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, MO(1),
    KC_LGUI, KC_LALT,                            KC_SPC,                                      KC_RALT,          KC_RCTL
    ),
    /* FN */
    [1] = LAYOUT_60_hhkb(
        KC_GRV,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_INS,  KC_DEL,
    KC_CAPS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_PSCR, KC_SLCT, KC_PAUS, KC_UP,   KC_TRNS, KC_TRNS,
    KC_TRNS, KC_VOLD, KC_VOLU, KC_MUTE, KC_EJCT, KC_TRNS, KC_PAST, KC_PSLS, KC_HOME, KC_PGUP, KC_LEFT, KC_RGHT, KC_TRNS,
    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_PPLS, KC_PMNS, KC_END,  KC_PGDN, KC_DOWN, KC_TRNS, KC_TRNS,
    KC_TRNS, KC_TRNS,                            KC_TRNS,                                     KC_TRNS,          KC_TRNS
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by zvecr — APPROVED** [review]

**Pull request merged**